docs-src/src/pages/components/FlashTables/FlashProps.js

```diff
@@ -0,0 +1,164 @@
+import React, {Component} from 'react'
+import Table from 'harmonium/lib/Table'
+import Layout from '../../../layouts/index.js'
+
+export default function FlashProps() {
+  return (
+    <Table>
+      <Table.Head>
+        <Table.Row>
+          <Table.Header>Name</Table.Header>
+          <Table.Header>Type</Table.Header>
+          <Table.Header>Default</Table.Header>
+          <Table.Header>Description</Table.Header>
+        </Table.Row>
+      </Table.Head>
+      <Table.HeadStacked>
+        <Table.Data>Flash Props</Table.Data>
```

Comment:
this needs to be changed to `<Table.Header>`

Reply:
I used the table layout template from the contributing.md file and it is setup the same way. Not exactly sure what you want me to change to a table.header? headstacked? or table.data? or table.head?